poker-pals/src/components/pokerComponents/Chat.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+    <div class="chatBox" :style="{ height: heightPercentages.chatBox }"> <!--the chat container holding all the chat elements together-->
+            <div class="title" :style="{ height: heightPercentages.title }">
+                <div class="title_text centerText">Big Fish</div>
```

Comment:
I think this hardcoded string should be extracted to the data object

Reply:
added to trello